lib/smart_proxy_ansible/api.rb

```diff
@@ -1,14 +1,16 @@
+require 'foreman_ansible_core'
```

Comment:
This is the key change - when the proxy is running on:

* Debian: foreman_ansible_core will load everything including the runner
* RH: foreman_ansible_core will only load the roles reader.